Directory.Build.props

```diff
@@ -1,28 +1,10 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <LibraryRoot>$(MSBuildThisFileDirectory)</LibraryRoot>
-    <LibraryToolsFolder>$(LibraryRoot)tools</LibraryToolsFolder>
-    <BuildAssetsDir>$(LibraryToolsFolder)\BuildAssets</BuildAssetsDir>
-    <SdkBuildToolsDir>$(LibraryToolsFolder)\SdkBuildTools</SdkBuildToolsDir>
+    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
+    <RepoEngPath>$(MSBuildThisFileDirectory)eng/</RepoEngPath>
```

Comment:
@weshaggard maybe this is a style issue
Can we not add trailing directory separator for root level properties/paths
for e.g. rather than adding trailing directory separator for <RepoEngPath>, use the directory separator below
$(RepoEngPath)/Directory.Build.Mgmt.props

Here are my reasons for it
When I use a directory separator, I know for sure that $(RepoEngPath) is a directory and not some other value.